app/src/components/common/PinExtensionPopup/PinExtensionPopup.css

```diff
@@ -0,0 +1,157 @@
+.pin-extension-popup-container .rq-modal.ant-modal {
+  width: 373px;
+  min-width: 370px !important;
+  margin: 0px;
+  top: 16px;
+  left: calc(100% - 373px - 16px); /* whole screen - container width - spacing */
+  transform-origin: top right !important;
+}
```

Comment:
Give a `className` to RQModal and add styles to it, instead of overwriting

Reply:
We can remove `width` from here as the same is given as `width` prop to RQModal. Also why do we need` min-width` here?

Reply:
This `min-width` is overriding the default RQModals min-width which is 550px